YAXLibTests/SerializerOptionsTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
+// Licensed under the MIT license.
+
+using System.Xml.Linq;
+using NUnit.Framework;
+using YAXLib;
+using YAXLib.Options;
+
+namespace YAXLibTests
+{
+    [TestFixture]
+    public class SerializerOptionsTests
+    {
+        private SerializerOptions GetDefaultOptions()
+        {
+            return new();
+        }
+
+        private SerializerOptions GetNonDefaultOptions()
+        {
+            return new() {
+                ExceptionBehavior = YAXExceptionTypes.Ignore,
+                ExceptionHandlingPolicies = YAXExceptionHandlingPolicies.DoNotThrow,
+                AttributeName = new YAXAttributeName
+                    {Dimensions = "test-dims", RealType = "test-realtype"},
+                Namespace = new YAXNameSpace
+                    {Prefix = "testyaxlib", Uri = XNamespace.Get("https://test.sinairv.com/yaxlib/")},
+                MaxRecursion = 123,
+                SerializationOptions = YAXSerializationOptions.SuppressMetadataAttributes
+            };
+        }
+
+        [Test]
+        public void Get_Set_All_Options_Test()
```

Comment:
I'm doubting if this test is really useful. Why should give GetDefaultOptions the opposite results of GetNonDefaultOptions?

Reply:
will remove